pkg/shadow/config.go

```diff
@@ -3,27 +3,18 @@ package shadow
 import (
 	"os"
 	"path/filepath"
-	"strconv"
-
 	"github.com/darwinia-network/shadow/pkg/shadow/util"
 )
 
 const (
-	ETHEREUM_ROPSTEN     string = "ETHEREUM_ROPSTEN"
-	ETHEREUM_RPC         string = "ETHEREUM_RPC"
-	SHADOW_GENESIS       string = "SHADOW_GENESIS"
-	GETH_DATADIR         string = "GETH_DATADIR"
 	DEFAULT_ETHEREUM_RPC string = "https://mainnet.infura.io/v3/0bfb9acbb13c426097aabb1d81a9d016"
```

Comment:
```DEFAULT_ETHEREUM_RPC ``` need to be removed too, and removed the code using ```DEFAULT_ETHEREUM_RPC ``` in header.go as well.

Reply:
removed (https://github.com/darwinia-network/shadow/commit/8c21c2506683b2d98c82c0ddba85c0d08786b354)